app/views/forecast_snapshots/show.html.erb

```diff
@@ -229,7 +229,7 @@
           alignTicks: false
         },
         title: {
-          text: 1 + i + '. ' + '<a href="http://data.uhero.hawaii.edu/#/series?id=' + chart_data[i].id + '" target="_blank">' + chart_data[i].title + ' (' + chart_data[i].series + ')' + '</a>',
+          text: i + '. ' + '<a href="http://data.uhero.hawaii.edu/#/series?id=' + chart_data[i].id + '" target="_blank">' + chart_data[i].title + ' (' + chart_data[i].series + ')' + '</a>',
```

Comment:
OK, I think that's done by my latest changes. Pls review, and if it's still not quite right, then I'm not sure where to go next...